gramine/README.md

```diff
@@ -0,0 +1,76 @@
+## How to run demo with Gramine
```

Comment:
Can you please provide a single script for app & post boot against SPS instead of this readme?